roles/chameleon_usage/tasks/usage_report.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Suggest we rename this to `user_project_report.yml` (as above suggestion). Or something... but they should be named the same, so it's more obvious what this is, precisely. This role contains several reports, all of which technically are "usage reports", a bit of clarity will greatly help future readers of the code.